docs/evm/cadence/batched-evm-transactions.md

```diff
@@ -0,0 +1,641 @@
+---
+title: Batched EVM Transactions Using Cadence
+sidebar_label: Batched EVM Transactions
+sidebar_position: 6
+---
+
+Integrating Cadence into EVM applications on Flow enables developers to leverage the best of both worlds.
+This guide demonstrates how to batch EVM transactions using Cadence, allowing applications to embed multiple EVM
+transactions in a single Cadence transaction, conditioning final execution on the success of all EVM transactions.
+
+This feature can supercharge your EVM application by unlocking experiences otherwise impossible on traditional EVM
+platforms.
+
+## Objectives
+
+After completing this guide, you'll be able to
+
+- Construct a Cadence transaction that executes several EVM transactions such that if any EVM transaction fails, the
+  entire set will revert
+- Read and write from smart contract functions on [EVM Flowscan](https://evm.flowscan.io/)
```

Comment:
Do you mind converting these to reference-style links?  It makes it much easier to read the raw content, and more practicable to link to the same thing multiple times.

Reply:
Updated in [465bc68](https://github.com/onflow/docs/pull/983/commits/465bc68a75459e244299f20e14d15102d80a9610)